src/screens/FeedScreen/ContentPoll.js

```diff
@@ -160,7 +161,7 @@ const stylesComponent = (pollLength) =>
       fontFamily: fonts.inter[600],
       fontWeight: 'bold',
       fontSize: normalizeFontSize(14),
-      color: COLORS.black
+      color: colors.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.black` to `colors.black`. Make sure that `colors.black` is defined and is the intended color.
```diff
- color: COLORS.black
+ color: colors.black
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->